CHANGELOG.rst

```diff
@@ -7,6 +7,77 @@ The format is based on `Keep a Changelog`_,
 and this project adheres to `Semantic Versioning`_.
 
 
+Version 0.15.0 (2022-08-05)
+---------------------------
+
+* Added: ``audformat.MiscTable``
+  which can store data
+  not associated with media files
+* Added: store scheme labels in a misc table
+* Added: dictionary ``audformat.Database.misc_tables``
+  holding misc tables of a database
+* Added: ``audformat.utils.difference()``
+  for finding index entries
+  that are only part of a single index
+  for a given sequence of indices
+* Added: ``audformat.utils.is_index_alike()``
+  for checking if a sequence of indices
+  has the same number of levels,
+  level names,
+  and matching dtypes
+* Added: ``audformat.define.DataType.OBJECT``
+* Added: ``audformat.utils.set_index_dtypes()``
+  to change dtypes of an index
+* Added: ``audformat.testing.add_misc_table()``
+* Changed: ``audformat.Database.update()``
+  can now join schemes
+  with different labels
+* Changed: ``audformat.utils.union()``,
+  ``audformat.utils.intersect()``,
+  and ``audformat.utils.concat()``
+  now support any kind of index
```

Comment:
I think we also need to mention that we convert any integer type to `Int64`

Reply:
This is not really a change.
Before integer dtype was not supported.

Reply:
Ah yes, that's true. But what we dropped is the special treatment when providing a mix of `filewise` and `segmented` index objects.

Reply:
Good point, I tried to summarize this in an additional point now with:

```rst
* Changed: ``audformat.utils.intersect()``
  no longer removes segments
  from a segmented index
  that are contained
  in a filewise index
```